src/pages/service_grocery.js

```diff
@@ -0,0 +1,19 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+
+import Page from 'components/layouts/Page';
+import GroceryFormWhere from 'containers/GroceryFormWhere';
+import GroceryFormWhen from 'containers/GroceryFormWhen';
+import GroceryFormWhat from 'containers/GroceryFormWhat';
```

Comment:
could we rename these to something like - `GroceryFormLocation` `GroceryFormContact` `GroceryFormItems` instead of using where/when/what?

Reply:
👍

Reply:
Went with `GroceryFormLocation`, `GroceryFormDate`, `GroceryFormItems`.